### PR TITLE
read s3 bucket region sent by server

### PR DESCRIPTION
**Pull request by ydayagi**
https://issues.redhat.com/browse/ECOPROJECT-272
Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

### CODE REVIEW

File: internal/datatransfer/s3/sync.go [hunk 1]
```diff
@@ -5,18 +5,18 @@ import (
 	"crypto/x509"
 	"encoding/base64"
 	"fmt"
+	"net/http"
```
Comment (pkliczewski) [inline internal/datatransfer/s3/sync.go:8]: Why do we need this import?

Comment (masayag) [reply internal/datatransfer/s3/sync.go:8]: this is just re-organizing the imports. not new addition (moved from line 14)

**Review by masayag — APPROVED** [review]

File: internal/datatransfer/s3/sync.go (hunk 1 above)
Comment (pkliczewski) [reply internal/datatransfer/s3/sync.go:8]: ah, I missed it.

**Review by pkliczewski — APPROVED** [review]

**Comment by pkliczewski** [comment]
@jakub-dzon please review

**Review by jakub-dzon — APPROVED** [review]

File: internal/datatransfer/s3/sync.go (hunk 1 above)
Comment (ydayagi) [reply internal/datatransfer/s3/sync.go:8]: @pkliczewski @masayag vscode's GO extension uses all kind of GO tools (e.g. gopls, golint) and they do it automatically.

**Pull request merged**